refarch-backend/src/main/java/de/muenchen/refarch/MicroServiceApplication.java

```diff
@@ -1,34 +1,18 @@
 package de.muenchen.refarch;
 
 import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
-import org.springframework.boot.autoconfigure.domain.EntityScan;
-import org.springframework.context.annotation.ComponentScan;
-import org.springframework.context.annotation.Configuration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 /**
  * Application class for starting the microservice.
  */
-@Configuration
-@ComponentScan(
-        basePackages = {
-                "org.springframework.data.jpa.convert.threeten",
-                "de.muenchen.refarch"
-        }
-)
-@EntityScan(
-        basePackages = {
-                "org.springframework.data.jpa.convert.threeten",
-                "de.muenchen.refarch"
-        }
-)
 @EnableJpaRepositories(
```

Comment:
Can you try to remove this annotation as well? If its not working, at least removing the package reference should work, right?

Reply:
from the official docs: 

`By default, Spring Boot enables JPA repository support and looks in the package (and its subpackages) where @SpringBootApplication is located. If your configuration has JPA repository interface definitions located in a package that is not visible, you can point out alternate packages by using @EnableJpaRepositories and its type-safe basePackageClasses=MyRepository.class parameter.`

So i removed the annotation, and i also remove the package from the `SpringBootApplication` Annotations, looks like we also dont need that one

Reply:
Great. :)